code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -215,7 +215,7 @@
 		else if(check_zone(M.zone_selected) == "head")
 			M.visible_message("<span class='notice'>[M] gives [src] a pat on the head to make [p_them()] feel better!</span>", \
 						"<span class='notice'>You give [src] a pat on the head to make [p_them()] feel better!</span>")
-			if(dna && dna.species && ((dna.features["tail_lizard"] != "None") || (dna.features["tail_human"] != "None") || ("mam_tail" in dna.species.mutant_bodyparts)))
+			if(dna && dna.species && ((src.dna.features["tail_lizard"] != "None") && (src.dna.features["tail_human"] != "None") && (src.dna.features["mam_tail"] != "None")))
```

Comment:
Already realized that, tbh. just changing from mam_tail from `in dna.species.mutant_bodyparts` to != "None" "fixed" the headpat

I was simply trying to overcome it before giving up because going home from work.